src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/CheckboxesContainerComponent.tsx

```diff
@@ -1,26 +1,28 @@
+import React from 'react';
 import { FormControlLabel, FormGroup, FormLabel } from '@material-ui/core';
 import Checkbox, { CheckboxProps } from '@material-ui/core/Checkbox';
 import FormControl from '@material-ui/core/FormControl';
 import { makeStyles } from '@material-ui/core/styles';
-import * as React from 'react';
-import { AltinnAppTheme } from 'altinn-shared/theme';
-import classNames from 'classnames';
+import cn from 'classnames';
+
+import type { IComponentProps } from '..';
+import type { IOption } from 'src/types';
+
 import { renderValidationMessagesForComponent } from '../../utils/render';
 import { useAppSelector } from 'src/common/hooks';
-import { IComponentProps } from '..';
 
 export interface ICheckboxContainerProps extends IComponentProps {
   validationMessages: any;
```

Comment:
```suggestion
  validationMessages: IComponentValidations;
```

While we are touching the code, could we add an interface here? Would have to import as well.

Reply:
Good suggestion, Ill add the import as well :)